src/utilities/core-utils.test.ts

```diff
@@ -1,14 +1,89 @@
 import { CoreUtils } from "./core-utils";
 
 describe("CoreUtils", () => {
+    // -----------------------------------------------------------------------------------------
+    // #region enumToArray
+    // -----------------------------------------------------------------------------------------
+
     describe("enumToArray", () => {
-        test.skip("TODO: Backfill tests to complete issue https://github.com/AndcultureCode/AndcultureCode.JavaScript.Core/issues/29", () => {});
+        test("when given enum, returns array of values", () => {
+            // Arrange
+            enum testEnum {
+                first = "First",
+                second = "Second",
+            }
+
+            // Act
+            const result = CoreUtils.enumToArray(testEnum);
+
+            // Assert
+            expect(result).toContain(testEnum.first);
+            expect(result).toContain(testEnum.second);
+        });
+
+        test("when given null enum, returns empty array", () => {
+            // Act
+            const result = CoreUtils.enumToArray(null);
+
+            // Assert
+            expect(result).toHaveLength(0);
+            expect(result).toBeInstanceOf(Array);
+        });
     });
 
+    // #endregion enumToarray
+
+    // -----------------------------------------------------------------------------------------
+    // #region getRandomEnum
+    // -----------------------------------------------------------------------------------------
+
     describe("getRandomEnum", () => {
-        test.skip("TODO: Backfill tests to complete issue https://github.com/AndcultureCode/AndcultureCode.JavaScript.Core/issues/29", () => {});
+        test("when given enum, returns random value", () => {
+            // Arrange
+            enum testEnum {
+                first = "First",
+                second = "Second",
+                third = "Third",
+            }
+            const expectedValues = [
+                testEnum.first,
+                testEnum.second,
+                testEnum.third,
+            ];
+
+            // Act
+            const result = CoreUtils.getRandomEnum(testEnum);
+
+            // Assert
+            expect(result).not.toBeNull();
+            expect(expectedValues).toContain(result);
+        });
+
+        test("when given enum and exclude element, returns random value and not excluded element", () => {
+            // Arrange
+            enum testEnum {
+                first = "First",
+                second = "Second",
+                third = "Third",
+            }
+            const expectedValues = [testEnum.first, testEnum.second];
+
+            // Act
+            const result = CoreUtils.getRandomEnum(testEnum, testEnum.third);
```

Comment:
I would also run this multiple times to make sure that it wasn't just lucky once.